### ISSUE TITLE
Missing tnum feature

### ISSUE THREAD

**Issue by kj** [TypeNetwork/rubik#11]
I've been 'following' this issue:

https://github.com/googlefonts/rubik/issues/22

Forgive me if I'm wrong, but it seems like the tnum feature is still missing from the latest fonts? I have been using Rubik in my status bar to show the current date, and it's frustrating that the numbers jump around. I'm very pleased to see there's still active development on this font!

**Comment by kj** [comment]
I've managed to come up with a temporary solution here (forked from the original repo):

https://github.com/kj/rubik/commit/1a1b8063c361cb18bacdfe1cce56eb6625da1773

I had to remove a glyph to get it to compile though, as I don't know enough about this to fix it properly. However, tnum is now working for the fonts built with this. I doubt this can be of much help here, as things seem to have changed significantly from the original googlefonts/rubik repo. It does at least prove the assumption in the issue I referenced above that the tnum glyphs are still there, just the feature wasn't defined, so I guess it's probably a similarly trivial fix in this updated fork.
